public/styles/style.css

```diff
@@ -73,4 +73,11 @@
     #blacklist button {
         width: 100%;
     }
+}
+
+#login {
```

Comment:
I'd just do #login section { margin: auto } instead, is better for responsiveness

Reply:
changed margin